doc/setup_instructions.md

```diff
@@ -338,39 +266,90 @@ If it didn't work you can try these:
 to get help. Links to the forums and community discord
 are at the top of this document.
 
-Forking
--------
+Git LFS limitations
+-------------------
 
-If you are just starting out and you don't have write access to the
-Thrive repository yet, you will need to create a fork. Use the fork
-button on Github on the main repository page. This will give you a
-fork under your github account (for example my fork is https://github.com/hhyyrylainen/Thrive).
+Due to current [Git LFS server](https://github.com/Revolutionary-Games/ThriveDevCenter)
+limitations you can't commit changes to files tracked by Git LFS if
+you are not a team member. If that is needed for a PR please ask some
+team member to commit the assets for you.
 
-Then you can add that to your git config in the thrive folder with this command:
+For information on committing to the LFS repository you can read this
+wiki page https://wiki.revolutionarygamesstudio.com/wiki/Git_LFS 
 
-```
-git remote add fork https://github.com/YOURUSERNAMEHERE/Thrive.git
-```
+Optional downloads
+------------------
+
+### Ruby
 
-Then after committing your changes (git-cola is recommended or another
-graphical tool for reviewing the exact lines you have changed, but you
-can also commit on the command line) you can publish them to your fork
-with (assuming you used the master branch, when working with the main
-thrive repository you MUST create a different branch but when working
-with a fork that isn't required, but still strongly recommended as
-making multiple pull requests with one branch is messy):
+Ruby is needed for the scripts to package the game, and for the
+code style checks. So while it is not necessary to download this
+it is highly reccomended.
 
+On Linux and mac you probably already have this, but if not, use a
+package manager to install it.
+
+On windows it is recommended to use RubyInstaller, version 2.4 or
+newer, when installing make sure to also install the MSYS option in
+order to be able to install gems.
+
+After installing ruby open a terminal / command prompt and run:
+
+```sh
+gem install os colorize rubyzip json sha3
```

Comment:
There's probably a merge conflict with this, as I added one more ruby gem to this list.